cmd/serf/command/agent/config.go

```diff
@@ -154,10 +162,12 @@ type Config struct {
 	// allows Serf agents to join each other with zero configuration.
 	Discover string `mapstructure:"discover"`
 
+	MDNS MDNSConfig `mapstructure:"mdns"`
+
 	// Interface is used to provide a binding interface to use. It can be
 	// used instead of providing a bind address, as Serf will discover the
 	// address of the provided interface. It is also used to set the multicast
-	// device used with `-discover`.
+	// device used with `-discover`, if mdns.interface is not set
```

Comment:
nit:
```suggestion
	// device used with `-discover`, if `mdns-iface` is not set
```

Reply:
Thanks a lot for your review. I added your suggestion in the amended commit